guides/17_Migration_to_3_0.md

```diff
@@ -159,13 +159,102 @@ handle them please remove.
 - StateMachineTransitionFailed
 - AudioDeviceSwitched
 
+## `MeetingSessionPOSTLogger`
+
+We have renamed `MeetingSessionPOSTLogger` to `POSTLogger` and removed the `MeetingSessionConfiguration` dependency. You don't need to pass the `MeetingSessionConfiguration` object to the `POSTLogger` constructor anymore.
+
+You can pass below `POSTLoggerOptions` to the `POSTLogger` constructor:
+```typescript
+interface POSTLoggerOptions {
+  url: string;
+  batchSize?: number;
+  headers?: Record<string, string>;
+  intervalMs?: number;
+  logLevel?: LogLevel;
+  metadata?: Record<string, string>;
+}
+```
+Please check `POSTLoggerOptions` documentation for more information.
+
+### Constructor
+
+```js
+// You need responses from server-side Chime API. See below for details.
+const meetingResponse = // The response from the CreateMeeting API action.
+const attendeeResponse = // The response from the CreateAttendee API action.
+
+// Before in 2.x
+const meetingSessionConfiguration = new MeetingSessionConfiguration(meetingResponse, attendeeResponse);
+const meetingSessionPOSTLogger = new MeetingSessionPOSTLogger(
+  'SDK',
+  configuration,
+  20, // LOGGER_BATCH_SIZE
+  2000, // LOGGER_INTERVAL_MS
+  URL,
+  LogLevel.INFO
+);
+
+// After in 3.x
+const options: POSTLoggerOptions = {
+  url: 'URL TO POST LOGS',
+  logLevel: LogLevel.INFO,
+  metadata: {
+    appName: 'SDK',
+    meetingId: meetingResponse.Meeting.MeetingId,
+    attendeeId: attendeeResponse.Attendee.AttendeeId,
+  },
+};
+const logger = new POSTLogger(options);
+```
+
+### Set `metadata` after `POSTLogger` creation
```

Comment:
How about just adding this code to the example above?
- Setting `metadata` will also help client applications with no UI frameworks.
- I think the provider and hook instruction can be into a Chime SDK React Component Library doc.

```js
// You can also set new metadata after initializing POSTLogger.
// For example, you can set metadata after receiving API responses from your server application.
logger.metadata = {
  appName: 'Your app name',
  meetingId: meetingResponse.Meeting.MeetingId,
  attendeeId: attendeeResponse.Attendee.AttendeeId,
};
```

Reply:
Thanks for the concise write-up have replaced existing with your provided write-up.